modules/plugin/jdbc/jdbc-oracle/src/main/java/org/geotools/data/oracle/OracleNGDataStoreFactory.java

```diff
@@ -21,11 +21,17 @@
 import java.sql.SQLException;
 import java.util.Collections;
 import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp.BasicDataSource;
```

Comment:
this warrants adding the apache commons dbcp library to the dependencies if the module